src/yellow/faq-connectivity/faq-is-z2m-suported.md

```diff
@@ -6,13 +6,7 @@ zendesk:
   labels: yellow, faq, connectivity
 ---
 
-The Home Assistant Yellow team officially supports [ZHA](https://www.home-assistant.io/integrations/zha/) (Zigbee Home Automation), the Zigbee stack integrated into [Home Assistant Core](https://www.home-assistant.io/docs/glossary/#home-assistant-core).
-
-The [Zigbee2MQTT](https://www.zigbee2mqtt.io/) project has [experimental support](https://www.zigbee2mqtt.io/guide/adapters/#experimental) (in beta stage) for the Silicon Labs radio on Home Assistant Yellow. With this setting, Zigbee2MQTT is known to work on Home Assistant Yellow.
-
-**Note**: The Zigbee Home Automation integration is active by default! If you are using Zigbee2MQTT, remember to delete and ignore the discovered Zigbee Home Automation integration to avoid interference.
-
-**Note**: When setting up Zigbee2MQTT with Home Assistant Yellow, you need to configure the adapter type in Zigbee2MQTT. Make sure the following is present in your Zigbee2MQTT [configuration file](https://www.home-assistant.io/docs/glossary/#configuration-file):
+{% include "connect-zbt-1/faq/is-z2m-supported.md" %}
 
 ```yaml
 serial:
```

Comment:
Z2M added a new driver named `ember` a while ago, this should be used nowadays.

I am not sure if hardware flow control (`rtscts: true`) works nowadays, the discussion in https://github.com/Koenkk/zigbee-herdsman/issues/943 is a bit inconclusive. Maybe @puddly can chime in on this one. But technically, `rtscts` should be set to true since our hardware supports hardware flow control and firmware should too.

Reply:
That's all, yes!

(If it helps, for the ZBT-2 we use `baudrate: 460800` and no `rtscts` config. Not sure if templating can handle that.)

Reply:
I excluded this section from reuse.